robot/Cumulus/tests/browser/zpayment_allocations/enable_pmt_alloc_and_test.robot

```diff
@@ -64,55 +90,50 @@ Update GAU Allocations and Verify Payment Allocations Sync
-    Checkbox Status    Payment Allocations Enabled    Checked
-    Checkbox Status    Default Allocations Enabled    Checked
-    Click Button    Edit
-    Choose Frame    Allocations Settings
-    Set Checkbutton To    Default_Allocations_Enabled    unchecked
-    Set Checkbutton To    Payment_Allocations_Enabled    unchecked
-    Populate Field With Id    Default__c    null
-    Click Button    Save      
+    Go To Page                      Custom                                  CustomSettings
+    Select Settings Option          Allocations Settings                    Manage
+    Verify Page And Select Frame    Allocations Settings
+    Wait Until Element Is Visible   text:Default Organization Level Value
+    Checkbox Status                 Payment Allocations Enabled             Checked
+    Checkbox Status                 Default Allocations Enabled             Checked
+    Click Button                    Edit
+    Verify Page And Select Frame    Allocations Settings
+    Set Checkbutton To              Default_Allocations_Enabled             unchecked
+    Set Checkbutton To              Payment_Allocations_Enabled             unchecked
+    Populate Field With Id          Default__c                              null
+    Click Button                    Save      
     
 Setup Test Data
     &{gau} =          API Create GAU  
     Set suite variable    &{gau}  
     ${date} =         Get Current Date    result_format=%Y-%m-%d
     Set suite variable    ${date}
-    &{contact} =      API Create Contact
-    Set suite variable    &{contact}
-    Store Session Record    Account    &{contact}[AccountId]
-    &{opportunity} =  API Create Opportunity   &{contact}[AccountId]    Donation  
-    ...    StageName=Prospecting    
-    ...    Amount=100    
-    ...    CloseDate=${date}    
-    ...    npe01__Do_Not_Automatically_Create_Payment__c=false    
-    ...    Name=&{contact}[LastName] Test Donation
+    Setupdata   contact1   ${contact1_fields}     ${opportunity1_fields}
+    Setupdata   contact2   ${contact2_fields}     ${opportunity2_fields}
+    @{records} =     Salesforce Query    npe01__OppPayment__c    
+    ...    select=Id
+    ...    npe01__Opportunity__c=${data}[contact1_opportunity][Id]
+    &{id} =     Get From List  ${records}  0
+    &{payment1} =     Salesforce Get  npe01__OppPayment__c  &{id}[Id]  
+    Set suite variable    &{payment1} 
     @{records} =     Salesforce Query    npe01__OppPayment__c    
     ...    select=Id
-    ...    npe01__Opportunity__c=&{opportunity}[Id]
+    ...    npe01__Opportunity__c=${data}[contact2_opportunity][Id]
     &{id} =     Get From List  ${records}  0
-    &{payment} =     Salesforce Get  npe01__OppPayment__c  &{id}[Id]  
-    Set suite variable    &{opportunity}
-    Set suite variable    &{payment} 
\ No newline at end of file
+    &{payment2} =     Salesforce Get  npe01__OppPayment__c  &{id}[Id]  
+    Set suite variable    &{payment2} 
+    Log   ${data}
```

Comment:
This log, may be you forgot to take out?

Reply:
yes thanks for catching

Reply:
removed